internal/oidc/kube_storage.go

```diff
@@ -183,26 +183,19 @@ func (k KubeStorage) RevokeRefreshToken(ctx context.Context, requestID string) e
 //
 // OAuth client definitions:
 //
-// For the time being, we only allow a single pre-defined client, so we do not need to interact with any underlying
-// storage mechanism to fetch them.
-//
 
-func (KubeStorage) GetClient(_ context.Context, id string) (fosite.Client, error) {
-	client := PinnipedCLIOIDCClient()
-	if client.ID == id {
-		return client, nil
-	}
-	return nil, fosite.ErrNotFound
+func (k KubeStorage) GetClient(ctx context.Context, id string) (fosite.Client, error) {
+	return k.clientManager.GetClient(ctx, id)
 }
 
 //
 // Unused interface methods.
```

Comment:
@mattmoyer Looks like you might have missed updating this comment?

Reply:
Ah thanks, fixed.